src/Cli/dotnet/BuiltInCommandsCatalog.cs

```diff
@@ -158,7 +159,11 @@ public static class BuiltInCommandsCatalog
             ["internal-reportinstallsuccess"] = new BuiltInCommandMetadata
             {
                 Command = InternalReportinstallsuccess.Run
-            }
+            },
+            ["workload"] = new BuiltInCommandMetadata
+            {
+                Command = WorkloadCommand.Run
```

Comment:
We'll also want to track adding these commands to the top level help once we remove the flag.